gatsby-config.js

```diff
@@ -134,6 +141,11 @@ module.exports = {
         theme_color: theme.COLORS_PRIMARY_500,
         display: `minimal-ui`,
         icon: 'content/assets/logos/roadie/roadie-r-round.svg',
+        // The request for the manifest was failing with a 401 on preview.roadie.io because
+        // of the basic authentication layer applpied to that site by Netlify. We use basic
+        // auth on the preview site to keep it hidden from search engines and (to a less
+        // important extent) people.
+        crossOrigin: `use-credentials`,
```

Comment:
This solves one of the 401s we encounter on preview.roadie.io. Explained here: https://thatemil.com/blog/2018/02/21/pwa-basic-auth/